content/docs/get_started/login_requiring.md

```diff
@@ -0,0 +1,163 @@
+              <amp-img width="44" class="user-avatar" height="44" alt="user" src="{{UserImg}}"></amp-img>
+              <div class="card comment">
+               <p><span class="user">{{User}}</span> <span class="date">{{Datetime}}</span></p>
+               <p>{{Text}}</p>
+              </div>
+            </div>
+      </template>
+</div>
+[/sourcecode]
+
+Notice how the template is wrapped in a div with the `submit-success` attribute. We are only checking if the value of the comment is not empty in this sample; in case it’s empty we return an error that would cause to execute the code:
+
+[sourcecode:html]
+<div submit-error>
+<template type="amp-mustache">
+            Error! Looks like something went wrong with your comment, please try to submit it again.
+      </template>
+</div>
+[/sourcecode]
+
+A little bit further on this, by using the attribute required, we are enforcing on the client side the presence of a text before hitting the Comment button:
+
+<amp-img src="/static/img/enforce-comment.png" alt="Enforce comment" height="325" width="300"></amp-img>
+
+[sourcecode:html]
+<input type="text" class="data-input" name="text" placeholder="Your comment..." required>
+[/sourcecode]
+
+Now add a comment and click comment, you should see something similar to the snapshot below:
+
+<amp-img src="/static/img/logout-button.png" alt="Comment added" height="352" width="300"></amp-img>
+
+## Logout
+Similar to the login button, the logout button is conditioned by the `amp-access` attribute:
+
+[sourcecode:html]
+<a amp-access="loggedIn" amp-access-hide role="button" tabindex="0" on="tap:amp-access.login-sign-out" class="button button-primary comment-button">Logout</a>
+[/sourcecode]
+
+The URL to be used once you click on the logout, is specified in the amp-access JSON configuration, as part of the login object:
+
+[sourcecode:json]
+{
+"login": {
+          "sign-in": "https://ampbyexample.com/samples_templates/comment_section/login?rid=READER_ID&url=CANONICAL_URL",
+          "sign-out": "https://ampbyexample.com/samples_templates/comment_section/logout"
+        }
+}
+[/sourcecode]
+
+Similar to the login, when the AMPByExample server receives a logout requests, it uses the return URL query parameter automatically added by the AMP library and redirects to it adding `#success=true`. By this time, you are back on the initial page; the AMPByExample cookie previously created for the login page (called `ABE_LOGGED_IN`) would be cleared at this point.
+
+
+## Conclusions
+In this tutorial we explored how to combine `amp-access` and `amp-form` to realize a comment section enabled by a login. We focused mostly on how we used those 2 components more than documenting them, while describing how we implemented a sample published on [ampbyexample.com.](https://ampbyexample.com/) In order to show the interaction between the component and a server, the sample provides a simple backend implementation which uses a cookie to save the login status.
```

Comment:
"We" should never mean you (the AMP team). In docs, "we" can either be used to include the reader, as in "now we, the group present, ..", or otherwise I'd just use you. If you want to explicitely talk about you, just use I. So I suggest rewriting this paragraph a little.